webpack.config.development.js

```diff
@@ -8,7 +8,13 @@ config.devServer = {
     ]
   },
   hot: true,
-  inline: true
+  inline: true,
+  proxy: {
+    '/api': {
+      changeOrigin: true,
+      target: 'http://localhost:3000'
```

Comment:
Yeah, I like that. I unfortunately already put up the [PR](https://github.com/smashingboxes/web-boilerplate/pull/46) up for it and it's closed. But I'll put in a separate PR for that. I like it.